### ISSUE TITLE
typescript-tools "Move to a new file" action is throwing an error

### ISSUE THREAD

**Issue by Skizy** [rachartier/tiny-code-action.nvim#4]
Hovering over action "Move to a new file" of typescript-tools (which uses tsserver as the lsp server) causes neovim to throw an error. This is the error when I'm calling `code_action()` over `rotateCube` function:
```
Error executing vim.schedule lua callback: Vim:E484: Cannot open file /home/skizy/projects/tests/rotateCube.js
stack traceback:                                                                                                                                          
        [C]: in function 'readfile'                                                                                                                       
        ...zy/tiny-code-action.nvim/lua/tiny-code-action/action.lua:94: in function 'preview_action_callback'                                             
        ...ny-code-action.nvim/lua/tiny-code-action/backend/vim.lua:24: in function 'callback'                                                            
        ...zy/tiny-code-action.nvim/lua/tiny-code-action/action.lua:20: in function 'handler'                                                             
        /usr/share/nvim/runtime/lua/vim/lsp/client.lua:687: in function ''                                                                                
        vim/_editor.lua: in function <vim/_editor.lua:0>
```
It looks like it's trying to open a non-existent file that would be created if I ran the action

**Comment by rachartier** [comment]
Does it works with "native" code actions ?

You can try with: `:lua vim.lsp.buf.code_action()`

**Comment by Skizy** [comment]
Yes, the action works as expected both with native code actions and with your plugin, it is just throwing an error when I'm hovering over it in the telescope window and it doesn't show the diff

**Comment by rachartier** [comment]
Okay, I'll need more infos: are you using `vim` or `delta` backend?
And can you send me a small "snippet" of code which can produce the error ? 

If so, thanks a lot !

**Comment by rachartier** [comment]
I've pushed a fix. It could resolve your issue

**Comment by Skizy** [comment]
I was using `vim` backend, and your fix did resolve the issue! Thanks for a very cool plugin!

**Comment by rachartier** [comment]
Thank you very much!

**Comment by SharpMars** [comment]
Hey, I installed this plugin yesterday and it looks like I'm having this issue but without any errors. When I open the menu it completely locks up nvim without any messages. Originally I used Telescope and it didn't show any menu and with snacks it shows the menu and the first option selected was "Move to a new file". When I filter it out it works correctly.

**Comment by rachartier** [comment]
Can you write me a code exemple, and which LSP do you use ?

**Comment by SharpMars** [comment]
[solid-project.zip](https://github.com/user-attachments/files/21907406/solid-project.zip)
Here's a repro, should happen on line 9 in App.tsx where `useQuery` is.

I'm using typescript-language-server from Mason:
```
typescript-language-server ts_ls
TypeScript & JavaScript Language Server.

installed version 4.4.0                                                                    
installed purl    pkg:npm/typescript-language-server@4.4.0                                 
homepage          https://github.com/typescript-language-server/typescript-language-server
```

And here are the relevant plugins:
```lua
{
  "mason-org/mason-lspconfig.nvim",
  opts = {},
  dependencies = {
    { "mason-org/mason.nvim", opts = {} },
    { "neovim/nvim-lspconfig" },
  },
  config = function()
    require("mason-lspconfig").setup {
      automatic_enable = {
        exclude = {
          "ts_ls"
        }
      }
    }
  end
},

{
  "rachartier/tiny-code-action.nvim",
  dependencies = {
    { "nvim-lua/plenary.nvim" },
    { "nvim-telescope/telescope.nvim" },
  },
  opts = { picker = "telescope", },
  event = "LspAttach",
  keys = {
    {
      "<leader>ca",
      function()
        require("tiny-code-action").code_action({
          --temporary fix
          --filter = function(action)
          --  return action.title:find("Move to a new file") == nil;
          --end
        })
      end,
      mode = { "n" },
      desc = "Code Action",
      noremap = true,
      silent = true,
    }
  }
},

{
  "pmizio/typescript-tools.nvim",
  dependencies = { "nvim-lua/plenary.nvim", "neovim/nvim-lspconfig" },
  opts = {},
  config = function()
    require("typescript-tools").setup {
      on_attach = function(client)
        client.server_capabilities.documentFormattingProvider = false
      end,
      settings = {
        complete_function_calls = true,
      }
    }
  end
},
```

**Comment by rachartier** [comment]
Should now be fixed in #76 . Can you please try and tell me if it works on your side?

**Comment by SharpMars** [comment]
Yup, works for me now. Now it just hits the timeout. Thanks
